include/GafferScene/EditScopeAlgo.h

```diff
@@ -135,6 +135,7 @@ enum class SetMembership
 	Unchanged
 };
 
+GAFFERSCENE_API Gaffer::Spreadsheet::RowPlug *acquireSetMembershipEdit( Gaffer::EditScope *scope, const std::string &set, bool createIfNecessary = true  );
```

Comment:
That does sound like it will be a little safer. Fixed in https://github.com/GafferHQ/gaffer/pull/5108/commits/3033e7223ac7706842eddb730c1f041222f33c2e and https://github.com/GafferHQ/gaffer/pull/5108/commits/d8d66ab284a999d3f28077ce2f3c581a348a8e41